api/src/main/java/org/eclipse/microprofile/config/ConfigAccessorBuilder.java

```diff
@@ -88,23 +88,23 @@
     ConfigAccessorBuilder<T> cacheFor(Duration duration);
 
     /**
-     * Whether to evaluate variables in configured values.
+     * Whether to expand variables in configured values.
      * A variable starts with '${' and ends with '}', e.g.
      * <pre>
      * mycompany.some.url=${myserver.host}/some/path
      * myserver.host=http://localhost:8081
      * </pre>
-     * If 'evaluateVariables' is enabled, the result for the above key
+     * If 'expandVariables' is enabled, the result for the above key
```

Comment:
Not sure about the name of expandVariables.

Reply:
I don't mind the name. The original PR was using either "evaluate variables" or "replace variables".
As long as we consistently use the same term through the code and spec, I'm fine with it.